postreise/plot/plot_interconnection_map.py

```diff
@@ -1,249 +1,179 @@
-import numpy as np
 import pandas as pd
-from bokeh.models import ColumnDataSource, HoverTool
 from bokeh.plotting import figure
-from bokeh.tile_providers import Vendors, get_provider
 from powersimdata.network.model import ModelImmutables
 from powersimdata.utility import distance
 
-from postreise.plot.plot_states import get_state_borders
-from postreise.plot.projection_helpers import project_borders, project_branch
+from postreise.analyze.check import _check_grid
+from postreise.plot.plot_states import plot_states
+from postreise.plot.projection_helpers import project_branch
 
 
 def count_nodes_per_state(grid):
     """Count nodes per state to add as hover-over info in :func`map_interconnections`
 
     :param powersimdata.input.grid.Grid grid: grid object.
-    :return: (*pandas.DataFrame*) -- dataframe containing state names and count of nodes per state.
+    :return: (*pandas.DataFrame*) -- index: state names, column: number of nodes.
```

Comment:
When I run this, what I'm getting back is a dataframe with two: columns `"state"` and `"count"`. Should we be setting `"state"` as the index and returning a Series instead?

Reply:
What do you prefer. Changing the docstring (two columns) or changing the code:
```
    state_counts = count_nodes_per_state(grid)
    plot_states(
        bokeh_figure=p,
        background_map=background_map,
        state_list=list(state_counts["state"]), --> state_counts.index
        labels_list=list(state_counts["count"]),
        tooltip_name="nodes",
    )
```

Reply:
I prefer to return a Series.
```python
state_counts = count_nodes_per_state(grid)
    plot_states(
        bokeh_figure=p,
        background_map=background_map,
        state_list=list(state_counts["state"]), --> state_counts.index
        labels_list=list(state_counts["count"]), --> state_counts.to_numpy()
        tooltip_name="nodes",
    )
```

Reply:
Done